### ISSUE TITLE
Carthage not building since "0.28.1"

### ISSUE THREAD

**Issue by walsht** [BranchMetrics/ios-branch-deep-linking-attribution#950]
If there is a bug or error in this repo, please file the issue here. 

When running carthage update I am getting this error below.  It works fine on "0.28.1"

```
*** Building scheme "Branch iOS SDK Carthage" in TestBed-Swift.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/tim/Git/app-techtoday/TechToday/Carthage/Checkouts/iOS-Deferred-Deep-Linking-SDK/Branch-TestBed-Swift/TestBed-Swift.xcworkspace -scheme Branch\ iOS\ SDK\ Carthage -configuration Release -derivedDataPath /Users/tim/Library/Caches/org.carthage.CarthageKit/DerivedData/11.1_11A1027/iOS-Deferred-Deep-Linking-SDK/0.29.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/0v/6qnqmgl5773fq57b8zjlw6yc0000gn/T/iOS-Deferred-Deep-Linking-SDK SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/tim/Git/app-techtoday/TechToday/Carthage/Checkouts/iOS-Deferred-Deep-Linking-SDK)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /Users/tim/Git/app-techtoday/TechToday/Carthage/build.log
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]
```

If you need assistance integrating a Branch SDK, please email support@branch.io. Thank you.

**Comment by echo-branch** [comment]
Could you try explicitly excluding Catalyst.  We don't support it yet.

Example
```
xcodebuild -scheme 'Branch iOS SDK Carthage' build -sdk iphoneos
xcodebuild -scheme 'Branch iOS SDK Carthage' build -sdk iphonesimulator
```

**Comment by paulo-depop** [comment]
@echo-branch I had to enable the new build system on TestBed-Swift.xcworkspace to make it work

**Comment by paulo-depop** [comment]
@echo-branch is a fix for this planned?

**Comment by echo-branch** [comment]
Yes.  I forgot to include it in the last patch release.  I'll be in the next one.

**Comment by walsht** [comment]
@echo-branch When will the next patch release?

**Comment by echo-branch** [comment]
Should be fixed in 0.30.0
